src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+
+import TabsItem from './TabsItem/TabsItem'
+import TabsText from './TabsText/TabsText'
+import { StyledCol, StyledNav, StyledDiv } from './Tabs.styled'
+// import TabsItems from './TabsItems'
+
+const Tabs = () => {
+	const TabsItems = [
```

Comment:
Tak to nie może działać. Jeśli chciałbyś mieć w aplikacji 2 taki zakładki (z różną treścią) to jak rozwiążesz ten problem? To muszą być dane pochodzące z zewnątrz czyli przez `props`.

PS. Możesz do tego wykorzystać `props.children`: https://react.dev/reference/react/Children